### DISCUSSION TITLE
Upload saves as well as download?

### DISCUSSION THREAD

**Discussion by Shades84** [ValvePython/steamctl#47] [Q&A]
As title suggests, is it possible to upload to the steam cloud as well as download?

**Comment by rossengeorgiev** [comment]
It can be implemented, but what is the use case for it.   
When will someone want to upload saves manually? Potentially overwriting their existing ones.

**Reply by Shades84** [reply]
The goal is to make a system that syncs some game saves to a phone or other device that doesnt have direct access to steam cloud. 

Think retroarch saves or any game that has a phone version and PC version. The app I make would run on a server, download the save data, sync to devices (using syncthing or similar) and then upload when they've been modified.

I know this introduces a LOT of points of failure, but i think it could be neat

**Reply by Shades84** [reply]
@rossengeorgiev what do you think of this idea?
